src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -35,15 +35,10 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
             "case", "if", "try", "do", "using"
         };
 
-        private static readonly IReadOnlyCollection<CompletionItem> KeywordCompletionItems = Keywords.Select(k => new CompletionItem
+        private static readonly IReadOnlyCollection<string> DesignTimeHelpers = new string[]
         {
-            Label = k,
-            InsertText = k,
-            FilterText = k,
-            Kind = CompletionItemKind.Keyword,
-            SortText = k,
-            InsertTextFormat = InsertTextFormat.Plaintext,
-        }).ToArray();
+            "__builder", "__o", "__RazorDirectiveTokenHelpers__", "BuildRenderTree"
```

Comment:
We should probably include a few of the [non-Blazor directive helpers](https://github.com/dotnet/aspnetcore/blob/2af2bc2fa9c46bc8ba32a5555e3310bee43f4a70/src/Razor/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/AttributeTargetingTagHelpers_DesignTime.codegen.cs) that aren't dynamic too:
- `__typeHelper`
- `__tagHelperExecutionContext`
- `__tagHelperRunner`